core/src/main/java/com/netease/arctic/io/ArcticHadoopFileIO.java

```diff
@@ -165,9 +172,14 @@ public boolean rename(String src, String dts) {
       Path dtsPath = new Path(dts);
       FileSystem fs = getFs(srcPath);
       try {
-        return fs.rename(srcPath, dtsPath);
+        if (fs.rename(srcPath, dtsPath) == false) {
+          throw new IOException("Fail to rename: from " + src + " to " + dts +
+              " and file system return false, need to check the hdfs path");
+        } else {
+          return true;
+        }
       } catch (IOException e) {
-        throw new UncheckedIOException("Failed to rename: from " + src + " to " + dts, e);
+        throw new UncheckedIOException("Fail to rename: from " + src + " to " + dts, e);
       }
     });
   }
```

Comment:
If we choose to throw an exception when fail to rename, this method should return `void` instead of return `boolean`.